### PR TITLE
feat(performance-metrics): Add metrics vital info (web vitals details) - [INGEST-545]

### PR DESCRIPTION
**Pull request by priscilawebdev**
- [x] Create VitalInfo Metrics

image

For follow-up PR's

- [ ]  Convert Chart to metrics
- [ ]  Write tests

### CODE REVIEW

**Review by matejminar — COMMENTED** [review]
Will this PR and https://github.com/getsentry/sentry/pull/30681 share the same API requests?

**Comment by priscilawebdev** [comment]
@matejminar yes :)

File: static/app/views/performance/vitalDetail/vitalDetailContent.tsx [hunk 1]
```diff
@@ -179,21 +183,60 @@ class VitalDetailContent extends React.Component<Props, State> {
...
+            project={projectIds}
+            environment={environment}
+            field={[field]}
+            query={new MutableSearch(query).formatString()} // TODO(metrics): not all tags will be compatible with metrics
+          >
+            {({loading: isLoading, response}) => {
+              const p75AllTransactions = response?.groups.reduce(
+                (acc, group) => acc + (group.totals[field] ?? 0),
+                0
+              );
```
[the remark this answers is not in the document]
Comment (k-fish) [reply static/app/views/performance/vitalDetail/vitalDetailContent.tsx:217]: Yeah math on percentiles won't be correct since it's based on population.

**Review by k-fish — APPROVED** [review]
Tentative +1, based on addressing the p75 thing (or choosing to leave it for the time being)

File: static/app/views/performance/vitalDetail/vitalDetailContent.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply static/app/views/performance/vitalDetail/vitalDetailContent.tsx:217]: will fix it in a follow-up PR...thanks 🙏

**Pull request merged**